tstyche.config.json

```diff
@@ -1,3 +1,4 @@
 {
-  "testFileMatch": ["packages/*/__typetests__/*.test.ts"]
+  "$schema": "https://tstyche.org/schemas/config.json",
+  "testFileMatch": ["packages/*/__typetests__/**/*.test.ts"]
```

Comment:
Apparently, match pattern was incorrect and did not include `packages/jest-types/__typetests__/expect/*.test.ts` files.